src/styles/utils/variables.scss

```diff
@@ -0,0 +1,11 @@
+$block-width: 100px;
+$block-height: 100px;
+$distance-between-block: 1px;
+$days:
+  "mon" 0,
+  "tue" $block-width + $distance-between-block,
+  "wen" 2 * ($block-width + $distance-between-block),
+  "thu" 3 * ($block-width + $distance-between-block),
+  "fri" 4 * ($block-width + $distance-between-block),
+  "sat" 5 * ($block-width + $distance-between-block),
+  "sun" 6 * ($block-width + $distance-between-block);
```

Comment:
you can calculate these values in loop

Reply:
I wish I could) I can't make it, I need your help how can I do this?
Another change was added.